samples/net/syslog_net/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _syslog-net-sample:
+
+Syslog net Application
+######################
+
+Overview
+********
+
+This sample application enables a remote syslog service that will
+send syslog messages to a remote server, as configured in ``prj.conf``.
+See https://tools.ietf.org/html/rfc5424 and https://tools.ietf.org/html/rfc5424
+for more details about syslog protocol over UDP.
+
+The source code for this sample application can be found at:
+:file:`samples/net/syslog_net`.
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+
+Building and Running
+********************
+
+Build syslog_net sample application like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/syslog_net
+   :board: <board to use>
+   :conf: <config file to use>
+   :goals: build
+   :compact:
+
+For configuring the remote IPv6 syslog server, set the following
```

Comment:
Do you need to make these changes to prj.conf before doing the build step?  If so, this should move before the build step above.